packages/yarnpkg-fslib/sources/algorithms/copyPromise.ts

```diff
@@ -51,8 +51,8 @@ export async function copyPromise<P1 extends Path, P2 extends Path>(destinationF
   }));
 }
 
-async function copyImpl<P1 extends Path, P2 extends Path>(prelayout: Operations, postlayout: Operations, updateTime: typeof FakeFS.prototype.utimesPromise, destinationFs: FakeFS<P1>, destination: P1, sourceFs: FakeFS<P2>, source: P2, opts: CopyOptions) {
-  const destinationStat = await maybeLStat(destinationFs, destination);
+async function copyImpl<P1 extends Path, P2 extends Path>(prelayout: Operations, postlayout: Operations, updateTime: typeof FakeFS.prototype.utimesPromise, destinationFs: FakeFS<P1>, destination: P1, sourceFs: FakeFS<P2>, source: P2, opts: CopyOptions, didParentExist: boolean) {
```

Comment:
I know that `copyImpl`'s parameters are currently a mess but we generally avoid having boolean parameters that aren't part of option bags in the codebase.

Reply:
I've changed it so the `didParentExist` argument is part of the options bag